server/tests/api/check-params/users.ts

```diff
@@ -901,7 +901,7 @@ describe('Test users API validators', function () {
       await makePostBodyRequest({ url: server.url, path: registrationPath, token: server.accessToken, fields })
     })
 
-    it('Should fail with a channel name that is the same than user username', async function () {
+    it('Should fail with a channel name that is the same as user username', async function () {
```

Comment:
is "user username" correct? sounds a bit strange

Reply:
You are totally right, I missed that one.